km62/Yevlentiev_Mykhailo/laboratory1.sql

```diff
@@ -1,2 +1,134 @@
 -- LABORATORY WORK 1
 -- BY Yevlentiev_Mykhailo
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці та видаляти з них дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER evlentiev IDENTIFIED BY love_sql
+DEFAULT TABLESPACE "USERS" TO evlentiev
+TEMPERARY TABLESPACE "TEMP" TO evlentiev
```

Comment:
A mistake in the word "TEMPERARY" need to write "TEMPORARY".
Superfluous "TO evlentiev" 
Forgotten ";"
Must be TEMPORARY TABLESPACE "TEMP";